docs/source/create/avatars/add-flow.rst

```diff
@@ -28,7 +28,7 @@ While experimenting, feel free to use `Mannequin with Hair <https://hifi-content
 Flow App
 --------
 
-Download and run the `Flow app <https://highfidelity.com/marketplace/items/370b8ff7-d5ad-4410-9c5c-0ead86e69756>`_ to configure your flow settings. 
+Download and run the `Flow app <https://docs.vircadia.com/_static/resources/script/flow/flowAppCpp/flowAppCpp.js>`_ to configure your flow settings.
```

Comment:
```suggestion
Load the `Flow app <https://docs.vircadia.com/_static/resources/script/flow/flowAppCpp/flowAppCpp.js>`_ into Vircadia to configure your flow settings. See :doc:`Load an Interface Script </script/interface-scripts>`
```
"Download and run" makes it sound like you just download it with your web browser and run it like any other executable on your computer.